packages/communication-react/CHANGELOG.md

```diff
@@ -11,7 +11,19 @@ Fri, 09 Jul 2021 20:41:33 GMT
 
 ### Changes
 
-- Bump react-components to v1.0.0-beta.2 ([PR #496](https://github.com/azure/communication-ui-library/pull/496) by jinan@microsoft.com)
+* Updated calling sdk dependency to 1.2.0-beta.1
```

Comment:
Use a consistent tense throughout, if possible use imperative tense throughout (so Update instead of Updates or Updated)